Assets/HoloToolkit/Speech/Scripts/TextToSpeechManager.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using UnityEngine;
+
+#if WINDOWS_UWP
+using Windows.Media.SpeechSynthesis;
+using Windows.Storage.Streams;
+using System.Threading.Tasks;
+#endif
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Enables text to speech using the Windows 10 <see cref="SpeechSynthesizer"/> class.
+    /// </summary>
+    /// <remarks>
+    /// <see cref="SpeechSynthesizer"/> generates speech as a <see cref="SpeechSynthesisStream"/>. 
+    /// This class converts that stream into a Unity <see cref="AudioClip"/> and plays the clip using 
+    /// the <see cref="AudioSource"/> you supply in the inspector. This allows you to position the voice 
+    /// as desired in 3D space. One recommended approach is to place the AudioSource on an empty 
+    /// GameObject that is a child of Main Camera and position it approximately 0.6 units above the 
+    /// camera. This orientation will sound similar to Cortanas speech in the OS.
+    /// </remarks>
+    public class TextToSpeechManager : Singleton<TextToSpeechManager>
```

Comment:
I'm fine with changing the design to what you propose if everyone feels that's a better design. I personally feel that 80% of the time users will want to set up the audio location once in the scene graph and then just pass text from that point forward.

We could make an override of the Speak method or even a static version of the Speak method that allows you to pass the synthesizer in as a parameter. I just don't feel that's the way this will be used most of the time.

Again, though, that's just my opinion. We can go with whatever the major consensus is.